staging/src/k8s.io/controller-manager/config/v1alpha1/conversion.go

```diff
@@ -18,8 +18,10 @@ package v1alpha1
 
 import (
 	"k8s.io/apimachinery/pkg/conversion"
+	cpconfig "k8s.io/cloud-provider/config"
+	serviceconfig "k8s.io/cloud-provider/service/config"
+	cmconfig "k8s.io/controller-manager/config"
 	"k8s.io/kube-controller-manager/config/v1alpha1"
```

Comment:
the v1alpha1 GenericControllerManagerConfiguration type should move to k8s.io/controller-manager as well so this dependency is dropped